.github/workflows/test_bundle.yml

```diff
@@ -0,0 +1,33 @@
+name: Test colcon-bundle (latest release) bundling navigation
+on:
+  pull_request: # used for testing the PR
+  schedule:
+    - cron: '0 * * * *'
+jobs:
+  bundle:
+    runs-on: ubuntu-latest
+    container:
+      image: rostooling/setup-ros-docker:ubuntu-xenial-ros-kinetic-ros-base-latest
+    steps:
+      # TODO(setup-ros-docker#7): calling chown is necessary for now
+      - name: setup-ros-docker#7 workaround
+        run: sudo chown -R rosbuild:rosbuild "$HOME" .
+      - uses: ros-tooling/action-ros-ci@0.0.17
+        id: action-ros-navigation
+        with:
+          package-name: ros_tutorials navigation
+          source-ros-binary-installation: kinetic
+          vcs-repo-file-url: "https://raw.githubusercontent.com/colcon/colcon-bundle/8732026918f8a8325ceb19bedc0adc95e431cd6b/.github/workflows/test_bundle.repos"
+      - uses: actions/setup-python@v1
+        with:
+          python-version: 3.5
+      - name: Install dependencies and colcon-bundle
+        run: |
+          sudo apt update
+          sudo apt -y install python3-apt python3-pip
+          sudo pip3 install -U setuptools==44.0.0
+          sudo pip3 install -U colcon-ros-bundle
```

Comment:
you could also install from pip so that you test master against master?
https://adamj.eu/tech/2019/03/11/pip-install-from-a-git-repository/

otherwise it may break when introducing incompatible changes before the release...

Reply:
I agree, let's catch breaking changes early.